build_scripts/build_opencv.sh

```diff
@@ -58,6 +58,7 @@ cmake -DCMAKE_BUILD_TYPE=RELEASE \
       -DINSTALL_TESTS=OFF \
       -DVIBRANTE=TRUE \
       -DWITH_CSTRIPES=OFF \
+      -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX} \
```

Comment:
OpenCV explicitly seeks for libs in these paths https://github.com/opencv/opencv/blob/master/CMakeLists.txt#L166.